File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/Services/DialogServiceDetailViewModel.cs
using DevExpress.Mvvm.POCO;

namespace MVVMDemo.Services {
    public class DialogServiceDetailViewModel {
        public string CustomerName { get; set; }

        public static DialogServiceDetailViewModel Create() {
            return ViewModelSource.Create(() => new DialogServiceDetailViewModel());
        }
        protected DialogServiceDetailViewModel() {

        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/Services/DialogServiceDetailViewModel.vb
Imports DevExpress.Mvvm.POCO

Namespace MVVMDemo.Services

    Public Class DialogServiceDetailViewModel

        Public Property CustomerName As String

        Public Shared Function Create() As DialogServiceDetailViewModel
            Return ViewModelSource.Create(Function() New DialogServiceDetailViewModel())
        End Function

        Protected Sub New()
        End Sub
    End Class
End Namespace
